docs/pages/guides/permissions.md

```diff
@@ -26,30 +28,40 @@ You can find the full list of available properties in [Apple's InfoPlistKeyRefer
 
 ### Bare workflow
 
 In the bare workflow, permissions are controlled in your project **AndroidManifest.xml**.
 
-Some Expo and React Native modules include permissions by default. If you use `expo-location`, for example, both the `ACCESS_COARSE_LOCATION` and `ACCESS_FINE_LOCATION` are implied and added to your app's permissions automatically. To limit the permissions your managed workflow app requires, add them them to a [list of explicitly excluded permissions](#excluding-android-permissions).
+Some Expo and React Native modules include permissions by default. For example, if you use `expo-location`, the `ACCESS_COARSE_LOCATION`, `ACCESS_FINE_LOCATION` and `FOREGROUND_SERVICE` are added to your app's permissions automatically. To limit the permissions your bare workflow app requires, see [Excluding Android Permissions](#excluding-android-permissions).
 
 Apps using dangerous or signature permissions without valid reasons _may be rejected by Google_. Make sure you follow the [Android permissions best practices](https://developer.android.com/training/permissions/usage-notes) when submitting your app.
 
-#### Excluding Android permissions
+### Excluding Android permissions
+
+When adding Expo and other React Native modules to your project, certain Android permissions might be added automatically. The modules should only add relevant permissions **required** to use the module. However, sometimes you may want to remove some of these permissions.
+
+When you are building your app with [EAS Build](/build/introduction/), you can exclude Android permissions by adding `android.blockedPermissions` in **app.json**. For more information, see the example in the [Android Permissions](/distribution/app-stores/#android-permissions) section in [Deploying to the App Stores](/distribution/app-stores).
+
+<ConfigClassic>
 
-When adding Expo and other React Native modules to your project, certain Android permissions might be implied automatically. The modules should only add relevant permissions **required** to use the module, however, sometimes you may want to remove some of these permissions.
+To limit the permissions your managed workflow app requires, set the `android.permissions` property in your **app.json** to list only the permissions you need, and Expo will also include the minimum permissions it requires to run.
+
+Learn how to configure permissions with the [app manifest `permissions` property](/versions/latest/config/app/#permissions).
+
+</ConfigClassic>
+
+<ConfigReactNative>
 
 Since the `android.permissions` manifest property doesn't work in the bare workflow, you'll need to edit **AndroidManifest.xml** to exclude specific permissions from the build. You can do that with the `tools:node="remove"` attribute on the `<use-permission>` tag.
```

Comment:
```suggestion
Since the `android.blockedPermissions` isn't applied in bare workflow (because **app.json** is not used to configure bare workflow projects), you'll need to edit **AndroidManifest.xml** to exclude specific permissions from the build. You can do that with the `tools:node="remove"` attribute on the `<use-permission>` tag.
```